code/controllers/subsystem/ticker.dm

```diff
@@ -266,6 +267,7 @@ SUBSYSTEM_DEF(ticker)
 
 	if(!CONFIG_GET(flag/ooc_during_round))
 		toggle_ooc(FALSE) // Turn it off
+		toggle_dooc(FALSE) // fuck off ghosts
```

Comment:
If I'm reading this right, it should toggle_dooc by default if there is no config; cross-ref
https://github.com/DDMers/NSV13/blob/d78798e9325925e0b47d84ff7091cdc282be2e2d/config/game_options.txt#L17

When you start up a new server with null configs, this means turning on global ooc will also enable dooc. Turning them on would require switching each one on independently, but is irrelevant as this server has config set. It won't have any effect on this server.

It might be more confusing for someone starting up their own server/testing/doing a new server, but it might also be useful.

Reply:
DOOC isn't a config option.

Reply:
It doesn't check dooc, it only checks if flag/ooc_during_round = null.

Reply:
Yes, and Dead OOC's check isn't even executed if OOC is disabled.

Reply:
Yes. And if you start a server with unset configs, it will individually disable both DOOC and OOC, requiring both to be turned on individually. It's a minor thing, and irrelevant if OOC during round config is set.

Reply:
Explaining this was more of a waste of my time then just deleting it.